setenv.ts

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs');
+const { writeFile, existSync } = require('fs');
+const { argv } = require('yargs');
+
+//check if .env file exists
+const envExist = fs.existsSync('.env');
+
+if (!envExist) { 
```

Comment:
We are checking here for actual `.env` file existence. What if that file is misconfigured.

What will happen if I have `.env` file, and that file si configured like so
```
SPRING_DATASOURCE_URL=jdbc:postgresql://localhost:5432/petclinic
SPRING_DATASOURCE_USERNAME=postgres
SPRING_DATASOURCE_PASSWORD=petclinic
```
> Notice that I did not provide variable `REST_API_URL`.

Will `REST_API_URL` have default value? Or will it be undefined?